Makefile

```diff
@@ -117,23 +117,27 @@ run-test-mode: generate fmt-check lint manifests ## Run against the configured K
 	go run -ldflags $(LDFLAGS) ./main.go -namespace=$(RUN_NAMESPACE) -dev -test-mode -webhook-port=0
 
 .PHONY: install
-install: manifests ## Install CRDs into a cluster
+install: $(KUSTOMIZE) manifests ## Install CRDs into a cluster
 	$(KUSTOMIZE) build config/crd | kubectl apply -f -
 
 .PHONY: uninstall
-uninstall: manifests ## Uninstall CRDs from a cluster
+uninstall: $(KUSTOMIZE) manifests ## Uninstall CRDs from a cluster
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
 .PHONY: deploy
-deploy: manifests ## Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+deploy: $(KUSTOMIZE) manifests ## Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 	cd config/manager && kustomize edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
-tools/bin/controller-gen: hack/tools/go.mod
-	go build -o $@ -modfile=$< sigs.k8s.io/controller-tools/cmd/controller-gen
+tools/bin/controller-gen:
+	cd hack/tools; go build -o ../../$@ sigs.k8s.io/controller-tools/cmd/controller-gen
```

Comment:
I think we could possibly do `$(abspath $@)` here to avoid all the relative path stuff, which would change if we ever moved the output.
There a similar opportunity with `../$<` in the manifests & generate rules.